Citrix/citrix-adc/tests/test_aaa.json

```
@@ -0,0 +1,38 @@
+{
+  "input": {
+    "message": "01/24/2025:12:06:37 GMT NetscalerCD07_1 0-PPE-0 : default AAA LOGIN_FAILED 73912122 0 : User testUser - Client_ip 1.2.3.4 - Failure_reason \"External authentication server denied access\" - Browser Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
+  },
+  "expected": {
+    "message": "01/24/2025:12:06:37 GMT NetscalerCD07_1 0-PPE-0 : default AAA LOGIN_FAILED 73912122 0 : User testUser - Client_ip 1.2.3.4 - Failure_reason \"External authentication server denied access\" - Browser Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
+    "event": {
+      "category": [
+        "network"
+      ],
+      "code": "LOGIN_FAILED",
+      "dataset": "audit_aaa",
+      "reason": "User testUser - Client_ip 1.2.3.4 - Failure_reason \"External authentication server denied access\" - Browser Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
```

I think the reason must be "External authentication server denied access"

```suggestion
      "reason": "External authentication server denied access,
```